ra/ra.go

```diff
@@ -396,7 +390,7 @@ var forbiddenMailDomains = map[string]bool{
 func validateEmail(address string) error {
 	email, err := mail.ParseAddress(address)
 	if err != nil {
-		return unparseableEmailError
+		return berrors.InvalidEmailError("%q is not a valid e-mail address", address)
```

Comment:
In #3146 I said:
> We should set a reasonable max # of characters to include from the received contact in the error message.

Do you think that's overkill? I can't remember if I had a specific reason to think that was needed when I filed the issue originally.

Reply:
Seems like slicing it to <254 chars should be sufficient: https://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690

Reply:
Oops, glossed over that, will do!